generator/vulkan/render.zig

```diff
@@ -95,7 +95,7 @@ const foreign_types = std.ComptimeStringMap([]const u8, .{
     .{ "RROutput", @typeName(c_ulong) },
     .{ "wl_display", "opaque {}" },
     .{ "wl_surface", "opaque {}" },
-    .{ "HINSTANCE", "std.os.HINSTANCE" },
+    .{ "HINSTANCE", "std.os.windows.HINSTANCE" },
     .{ "HWND", "*opaque {}" },
     .{ "HMONITOR", "*opaque {}" },
     .{ "HANDLE", "std.os.HANDLE" },
```

Comment:
Would you please also HANDLE to `std.os.windows.HANDLE`?

Reply:
I just did that now actually; I've also noticed that `HWND` and `HMONITOR` are `*opaque {}`; should I go ahead and make them reference the canonical windows declarations as well?

Reply:
@Snektron pinging you just in case you didn't get a notification for my previous comment

Reply:
For `HWND`, yes, but i don't think Zig has a definition for `HMONITOR`. If you want to go the extra mile you could also make a pull request to Zig, but otherwise you can just leave `HMONITOR` as opaque.

Reply:
@Snektron Alright, I might chew on that some time soon.